cyipopt/scipy_interface.py

```diff
@@ -270,6 +274,7 @@ def get_constraint_dimensions(constraints, x0):
 
 
 def get_constraint_bounds(constraints, x0, INF=1e19):
+    TYPE_EQ, TYPE_INEQ = 'eq', 'ineq'
```

Comment:
While this use of tuple unpacking is clever, I personally don't think it's the clearest to read. Can we please split to two assignments over two lines.